cargo-registry/src/crate_handler.rs

```diff
@@ -110,8 +110,11 @@ impl Program {
             return Err("Signer doesn't match program ID".into());
         }
 
-        let mut program_data = read_and_verify_elf(self.path.as_ref(), &client.rpc_client)
```

Comment:
Are we relying on deployment to verify ELF with this change? i.e. the deployment will fail on verification failure.

Reply:
The check was redundant as it happens inside `process_deploy_program()` anyway, which is subsequently called.